### PR TITLE
Adds Deer ;D

### PR DESCRIPTION
**Pull request by Nienhaus**
Deer are buyable.
@MrSnapwalk for the lovely sprite
lsxettn

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
Oh dear...

**Comment by Spacemanspark** [comment]
:+1:

**Comment by Nienhaus** [comment]
Dat deer doe.

**Comment by FalseIncarnate** [comment]
Now the wall mount suggestion REALLY needs to be done...

:+1:

File: code/modules/mob/living/simple_animal/friendly/deer.dm [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	desc = "A strong, brave deer."
+	icon_state = "deer"
+	icon_living = "deer"
+	icon_dead = "deer_dead"
+	speak = list("snorts")
+	speak_emote = list("snorts")
+	emote_see = list("shakes its head")
+	speak_chance = 1
+	turns_per_move = 5
+	see_in_dark = 0 //I'm so funny
```
Comment (Iamgoofball) [inline code/modules/mob/living/simple_animal/friendly/deer.dm]: Oh my god ahahahahahhahahahahaha

**Comment by CrAzYPiLoT-SS13** [comment]
FalseIncarnate
Now the wall mount suggestion REALLY needs to be done

I am working on it. It's not hard, but it's gonna be snowflakey as fuck
unless there's a way to put object overlays on other objects that I am not
aware of. For now I gotta make a separate sprite for every possible gun and
head. That's the reason why it's taking so long.

**Comment by veganzombeh** [comment]
Absolutely love the sprite :+1:

**Comment by monkeysfist101** [comment]
Who the Hell though it would be a good idea to bring a fully grown eight point onto a space  station? I got bit in the ass, it pissed on my workstation to mark its territory, it mutilated one of my interns with its antlers, and clubbed my Head of Staff to death with its hooves. All in all, I had a great time and grew as a person, so I'm only taking off one half star. 9/10 IGN

**Comment by Aurorablade** [comment]
Meanwhile at the bar ... https://www.youtube.com/watch?v=vZrQR6QgsiQ

**Comment by Fox-McCloud** [comment]
Deery me...

**Pull request merged**